packages/sources/bitgo-reserves/src/config/index.ts

```diff
@@ -0,0 +1,9 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const config = new AdapterConfig({
+  API_ENDPOINT: {
+    description: 'An API endpoint for Data Provider',
+    type: 'string',
+    default: 'https://d2nz84bmzl209r.cloudfront.net',
```

Comment:
I think we should not set the testing endpoint as default?

Reply:
so set the real one and overwrite with the test one? or just hold until the real one is ready?

Reply:
rather set the real one than blank